From 1260271ec7f653612afac58afc0cada864487e07 Mon Sep 17 00:00:00 2001
From: Song Zhao <song.zhao@amlogic.com>
Date: Tue, 15 Oct 2019 09:36:23 -0700
Subject: [PATCH 3/5] workaround: tee: disable microCode loading by optee

Change-Id: I944ae3052fff5e046f4a6e859bbf0bae1c9f02a8
Signed-off-by: Song Zhao <song.zhao@amlogic.com>
(cherry picked from commit ceccb8efde2a0d569db2b80fd84fac12fffaae08)
---
 drivers/amlogic/tee/tee.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/tee/tee.c b/drivers/amlogic/tee/tee.c
index 296cacef0afd..39f401f438b8 100644
--- a/drivers/amlogic/tee/tee.c
+++ b/drivers/amlogic/tee/tee.c
@@ -168,7 +168,7 @@ bool tee_enabled(void)
 	struct arm_smccc_res res;
 	if (disable_flag == 1)
 		return false;
-	/*return false;*/ /*disable tee load temporary*/
+	return false; /*disable tee load temporary*/
 
 	arm_smccc_smc(TEE_SMC_CALLS_UID, 0, 0, 0, 0, 0, 0, 0, &res);
 
-- 
2.24.1

